test/models/host_test.rb

```diff
@@ -163,6 +163,41 @@ def teardown
     end
   end
 
+  test "should trigger orchestration when importing facts if unattended_orchestration is true" do
```

Comment:
Please use `context` block, and `setup` and `teardown` for setting up the settings: the reason is, that if something fails during the the test method, the setting to the original value would not happen